deployments/examples/ocis_wopi/docker-compose.yml

```diff
@@ -104,7 +104,9 @@ services:
     image: owncloud/ocis:${OCIS_DOCKER_TAG:-latest}
     networks:
       ocis-net:
-    command: app-provider server
+    entrypoint:
+      - /bin/sh
+    command: ["-c", "curl -k --fail https://${COLLABORA_DOMAIN:-collabora.owncloud.test} || exit 1 && ocis app-provider server" ]
```

Comment:
looks quite similar to what we already have inhttps://github.com/owncloud/ocis/blob/master/deployments/examples/ocis_wopi/config/ocis-appprovider-onlyoffice/entrypoint-override.sh

Could that be reused?

Reply:
done